src/util/search-providers/imdb.ts

```diff
@@ -12,19 +12,26 @@ const IMDB_SEARCH_PARAMS = {
  * @param params - search parameters
  */
 export async function imdb(params: ISearchParameters): Promise<ISearchResult[]> {
-  let result;
 
   try {
-    result = await imdbAPI.search({ name: params.title, year: params.year }, IMDB_SEARCH_PARAMS);
+    const imdbID = /([tt])\w+/;
```

Comment:
What about this RegExp `/[t]{2}[0-9]+/g`? It should help to avoid false positives.

Reply:
06e3eabaa711675cc9422205cf0004d4b54c92dc